MatrixSDK/Room/Polls/PollAggregator.swift

```diff
@@ -52,6 +52,7 @@ public class PollAggregator {
     
     private var events: [MXEvent] = []
     private var hasBeenEdited = false
+    private var relatedErroredEvents: Set<String> = []
```

Comment:
Optional: `Errored` does seem to be grammatical, though I find it a bit unusual and almost looks like a typo. Maybe `Failed` would be better? Alternatively if this only captures decryption errors, it could even be `relatedUndecryptableEvents`

Reply:
Yeah, it capture just crypto errors.
`relatedUndecryptableEvents` looks better. ;-)

Reply:
Applied -> https://github.com/matrix-org/matrix-ios-sdk/pull/1673/commits/7a29bc6bc5f6972427c8f6c71af39c2fe051eba9